src/openvpnserv/interactive.c

```diff
@@ -1474,9 +1476,13 @@ UpdateWaitHandles (LPHANDLE *handles_ptr, LPDWORD count,
       if (pos == size)
         {
           size += 10;
-          handles = realloc (handles, size * sizeof (HANDLE));
-          if (handles == NULL)
-            return ERROR_OUTOFMEMORY;
+	  void * tmp = realloc (handles, size * sizeof (HANDLE));
+ 	  if (tmp == NULL)
+	  {
+		free(handles);
```

Comment:
This will lead to a double free when the caller frees handles_ptr which points to the previous value of handles.